src/resources/stubs/migrations/2014_02_04_000000_create_country_state_city_table.stub

```diff
@@ -17,6 +17,8 @@ class CreateCountryStateCityTable extends Migration
         Schema::create('countries', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->string('name',255);
+            $table->string('currency_code');
```

Comment:
add string length $table->string('currency_code',255);

Reply:
Thank you @dipeshsukhia,
I will make the change and request a new review.